lifelines/statistics.py

```diff
@@ -646,6 +675,18 @@ def multivariate_logrank_test(
     t_0: float, optional (default=-1)
         the period under observation, -1 for all time.
 
+    weightings: str, optional
+        apply a weighted logrank test: options are "w" for Wilcoxon (also known as Breslow), "tw"
```

Comment:
I feel these should be more explicit. Ex: `weightings="wilcoxon"` vs `weights="w"`. The latter feels like an R thing, and the former a Python thing (at least, how I want Python to be). Thoughts?

Reply:
Yes, that seems better.